experimental/healthsea_spancat/project.yml

```diff
@@ -0,0 +1,121 @@
+# haven't changed, it won't be re-run.
+workflows:
+  all:
+    - preprocess
+    - train_ner
+    - train_spancat
+    - evaluate
+  ner:
+    - preprocess
+    - train_ner
+    - evaluate_ner
+  spancat:
+    - preprocess
+    - train_spancat
+    - evaluate_spancat
+
+
+# Project commands, specified in a style similar to CI config files (e.g. Azure
+# pipelines). The name is the command name that lets you trigger the command
+# via "spacy project run [command] [path]". The help message is optional and
+# shown when executing "spacy project run [optional command] [path] --help".
+commands:
+  - name: "preprocess"
+    help: "Format .jsonl annotations into .spaCy training format for NER and Spancat"
+    script:
+      - "python scripts/preprocess.py assets/annotation.jsonl data/${vars.train}.spacy data/${vars.dev}.spacy ${vars.eval_split}"
+    deps:
+      - "assets/annotation.jsonl"
+      - "scripts/preprocess.py"
+    outputs:
+      - "data/${vars.train}.spacy"
+      - "data/${vars.dev}.spacy"
+
+  - name: "train_ner"
+    help: "Train an NER model"
+    script:
+      - "python -m spacy train configs/ner/${vars.ner_config}.cfg --output training/ner/${vars.ner_config}/ --paths.train data/${vars.train}.spacy --paths.dev data/${vars.dev}.spacy --gpu-id ${vars.gpu_id}"
+    deps:
+      - "data/${vars.train}.spacy"
+      - "data/${vars.dev}.spacy"
+      - "configs/ner/${vars.ner_config}.cfg"
+    outputs:
+      - ${vars.models.model_ner}
+
+  - name: "train_spancat"
+    help: "Train a Spancat model"
+    script:
+      - "python -m spacy train configs/spancat/${vars.spancat_config}.cfg --output training/spancat/${vars.spancat_config}/ --paths.train data/${vars.train}.spacy --paths.dev data/${vars.dev}.spacy --gpu-id ${vars.gpu_id}"
+    deps:
+      - "data/${vars.train}.spacy"
+      - "data/${vars.dev}.spacy"
+    outputs:
+      - ${vars.models.model_spancat}
+
+  - name: "evaluate_ner"
+    help: "Evaluate the trained NER model"
+    script:
+      - "python -m spacy evaluate ${vars.models.model_ner} data/${vars.dev}.spacy --output metrics/ner_${vars.ner_config}.json --gpu-id ${vars.gpu_id}"
+    deps:
+      - "${vars.models.model_ner}"
+      - "data/${vars.dev}.spacy"
+    outputs:
+      - "metric/ner_${vars.ner_config}.json"
+
+  - name: "evaluate_spancat"
+    help: "Evaluate the trained Spancat model"
+    script:
+      - "python -m spacy evaluate ${vars.models.model_spancat} data/${vars.dev}.spacy --output metrics/spancat_${vars.spancat_config}.json --gpu-id ${vars.gpu_id}"
+    deps:
+      - "${vars.models.model_spancat}"
+      - "data/${vars.dev}.spacy"
+    outputs:
+      - "metric/ner_${vars.spancat_config}.json"
+
+  - name: "evaluate"
+    help: "Evaluate NER vs Spancat on the dev dataset"
+    script:
+      - "python scripts/evaluation.py ${vars.spankey} ${vars.gpu_id} ${vars.models.model_ner} ${vars.models.model_spancat} data/${vars.dev}.spacy --verbose"
```

Comment:
On my system, this crashes when `vars.gpu_id` is `-1`, because Python tries to interpret it as an option:
```
================================== evaluate ==================================
Running command: '...\python.exe' scripts/evaluation.py health_aspects -1 training/ner/config_tok2vec/model-best training/spancat/config_tok2vec/model-best data/dev.spacy --verbose
Usage: evaluation.py [OPTIONS] SPAN_KEY GPU_ID NER_MODEL_PATH
                     SPANCAT_MODEL_PATH TEST_PATH
Try 'evaluation.py --help' for help.

Error: No such option: -1
```
It does work like a charm if I change `vars.gpu_id` to `0` for instance.